src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -315,6 +315,7 @@ private AccountAttributes createAccountAttributesToSanitize() {
                     .withGender(gender)
                     .withMoreInfo(moreInfo)
                     .withPictureKey(pictureKey)
+                    .withModifiedDate(Instant.parse("2018-03-30T13:23:42.663Z"))
```

Comment:
Can use `Const#TIME_REPRESENTS_DEFAULT_TIMESTAMP` if you need an arbitrary `Instant`.